sound/soc/sof/intel/cnl.c

```diff
@@ -280,6 +280,15 @@ int cnl_ipc4_send_msg(struct snd_sof_dev *sdev, struct snd_sof_ipc_msg *msg)
 	snd_sof_dsp_write(sdev, HDA_DSP_BAR, CNL_DSP_REG_HIPCIDR,
 			  msg_data->primary | CNL_DSP_REG_HIPCIDR_BUSY);
 
+	/* Don't need to trigger d0i3_work for pm setting which is defined in module msg */
```

Comment:
nit-pick: I'd say "No need to schedule d0i3 work when sending PM messages".

Reply:
sure,  much better, thanks!